ripd/rip_interface.c

```diff
@@ -225,39 +225,6 @@ static void rip_request_interface(struct interface *ifp)
 	}
 }
 
-#if 0
```

Comment:
You should not remove that code, let's keep it with #if 0 
This code is inteded to be called from rip_interface_add() when zebra provides a notification. It helps to have a faster convergence. It was not finished, so Stephen did set if between #if 0 in order to avoid a compilation warning.

Please, do not remove that code. If you want to fix it, it should be into another issue/pull request.